test/suite/test_bug018.py

```diff
@@ -82,17 +85,29 @@ def test_bug018(self):
             except wiredtiger.WiredTigerError:
                 self.conn = None
 
+    def test_bug018(self):
+        '''Test closing multiple tables'''
+
+        self.close_conn()
+        subdir = 'SUBPROCESS'
+        [ignore_result, new_home_dir] = self.run_subprocess_function(subdir,
```

Comment:
If some test assertion fails in the subprocess, is it going to fail the test?
I kind of assumed that you'd need to parse an exit code for the subprocess.

Reply:
@sueloverso, can you weigh in on this, since you wrote this test?  As I understand it, if we're closing a file handle underneath WiredTiger, we don't really care how it exits, or what error is returned, or even if an error is detected, what we care about is how recovery works in the resulting directory.  Is that correct?

Reply:
The test originally expected the an error, but not looking for a specific message. So a general test is probably sufficient.